source/content/dashboard.md

```diff
@@ -21,7 +21,7 @@ Easily visualize and manage sites, users, traffic, metrics, visual regression te
 
 Our site dashboard provide a real-time interface into Pantheon. It includes everything you need to build, launch, and run sites. Create new projects, manage team access, interface via SFTP or Git (version control), and track miscellaneous tools like backups. It also allows you to deploy and scale your sites.
 
-![The Pantheon Dashboard](../images/dashboard/new-dashboard/2024/site-dashboard.png)
+![The Pantheon Dashboard](../images/dashboard/new-dashboard/2024/_site-dashboard.png)
```

Comment:
Nit: would probably prefer replacing the old screenshot or renaming the new one (e.g. `site-dashboard-2024.png`) rather than adding an underscore. What happens with the next redesign? And the next? etc. It sets a precedent that I don't think we'd want to perpetuate in the future.

Reply:
I'll rename this to `../images/dashboard/new-dashboard/2024/site-dashboard-2024.png` as suggested

I don't feel strongly about the renaming convention so long as the name is changed then we'll get the fresh version. 

For background though, I added `2024` subdir to the path in an earlier PR which contained mostly the same images just with the nav cropped out based on PDEs recommendations, in a follow up PR I generated fresh screenshots from the product with the new nav shown for context and added `_` to the filenames because it was an easy modifier to do a search and replace for and I had already done the `2024` modifier previously in the folder structure